x-pack/plugins/ml/public/application/services/anomaly_explorer_charts_service.ts

```diff
@@ -381,8 +384,47 @@ export class AnomalyExplorerChartsService {
     return combinedJobs;
   }
 
+  public loadDataForCharts$(
+    jobIds: string[],
+    earliestMs: number,
+    latestMs: number,
+    influencers: EntityField[] = [],
+    selectedCells: AppStateSelectedCells | undefined,
+    influencersFilterQuery: InfluencersFilterQuery
+  ) {
+    if (
+      selectedCells === undefined &&
+      influencers.length === 0 &&
+      influencersFilterQuery === undefined
+    ) {
+      of([]);
+    }
+
+    return from(
+      this.mlResultsService.getRecordsForInfluencer(
```

Comment:
It would be great if you can make this method `getRecordsForInfluencer` to use Observables

Reply:
Updated here [`1968113` (#95623)](https://github.com/elastic/kibana/pull/95623/commits/1968113d887d64fd0dd7c5c02f1052a944e9fcb8)